android/daemon/build.gradle

```diff
@@ -29,38 +30,41 @@ buildscript {
 }
 
 plugins {
-    id "com.jetbrains.python.envs" version "0.0.26"
+    id "com.jetbrains.python.envs"
 }
 
-
 apply plugin: "com.android.library"
 apply plugin: "org.jetbrains.kotlin.android"
 apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
 
-
 android {
     compileSdkVersion Config.compileSdkVersion
     ndkVersion Config.ndkVersion
     defaultConfig {
         minSdkVersion Config.minSdkVersion
         targetSdkVersion Config.targetSdkVersion
-        
+
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        buildConfigField  "String", "VERSIONCODE" ,  '"' +System.getenv("VERSIONCODE") + '"'
+
+        buildConfigField  "String", "VERSIONCODE" ,  '"' + System.getenv("VERSIONCODE") + '"'
         buildConfigField  "String", "SHORTVERSION" ,  '"' + System.getenv("SHORTVERSION") +  '"'
```

Comment:
I think the reason why build config is not generated by glean_parser in here, is because the daemon is a library. Since the build info is required for initializing Glean and there is [a strong recomendation from Glean](https://mozilla.github.io/glean/book/user/adding-glean-to-your-project/index.html#glean-integration-checklist) not to initialize Glean on libraries, maybe that step is skipped? cc @badboy for confirmation if this is a bug or intended behaviour.